src/services/team/team.schema.ts

```diff
@@ -0,0 +1,83 @@
+// // For more information about this file see https://dove.feathersjs.com/guides/cli/service.schemas.html
+import { resolve } from '@feathersjs/schema'
+import { Type, getValidator, querySyntax } from '@feathersjs/typebox'
+import type { Static } from '@feathersjs/typebox'
+
+import type { HookContext } from '../../declarations'
+import { dataValidator, queryValidator } from '../../validators'
+import { toLowerCaseProperty } from '../../utilities/property-name-converter';
+
+
+// Main data model schema
+
+export const teamSchema = Type.Object(
+    {
+      id: Type.Number(),
+      name: Type.String(),
+      shortname: Type.String(),
+      logosrc: Type.String(),
+      photosrc: Type.String(),
+      description: Type.String(),
+      captainname: Type.String(), 
+      capitainemail:Type.String(), 
+      capitainphone: Type.String(), 
+    },
+    { $id: 'Team', additionalProperties: false }
+  )
+  
+
+export type Team = Static<typeof teamSchema>
+export const teamValidator = getValidator(teamSchema, dataValidator)
+export const teamResolver = resolve<Team, HookContext>({
+  photosrc: async (value) => {
+    if (value != null && value.trim() !== "") {
+      return `http://svcsa.org/uploads/${value}`;
+    }
+    return "No Image Uploaded!";
```

Comment:
should be return null? 
and make the photosrc to be optional?

Reply:
photosrc set to optional